.github/workflows/build-with-bal-test-native.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   call_stdlib_workflow:
     name: Run StdLib Workflow
     if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
-    uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-native-template.yml@main
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-graalvm-template.yml@main
```

Comment:
Need to rename the workflow file to - `build-with-bal-test-graalvm.yml`

Reply:
Renamed now.